scripts/sql/vis_dim_counts.sql

```diff
@@ -1,17 +1,19 @@
-create or replace PROCEDURE           visdim_counts( m_version IN VARCHAR
+icreate or replace PROCEDURE    visdim_counts( m_version IN VARCHAR
```

Comment:
it may have appeared when i copied from sql-developer and inserted into a vim editor. will remove it. this is the fixed procedure -  one i executed this AM sans the "i"